src/test/fuzz/netbase_dns_lookup.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <netbase.h>
```

Comment:
suggestion, as this uses `CNetAddr` defined in `netaddress.h`
```diff
+#include <netaddress.h>
```

Reply:
Thanks! Addressed.